tests/types/string.spec.ts

```diff
@@ -0,0 +1,59 @@
+import {StrongString, makeString} from '../../src/types/string';
+
+import {isType} from '../../src/validator/is-type';
+
+const MOCK_INITIAL = 'hello';
+const MOCK_FALLBACK_DEFAULT = 'hello world';
+const MOCK_FALLBACK = 'world';
+const EMPTY_STRING = '';
+
+describe('StrongString', () => {
+	describe('Implementation', () => {
+		it('should set initial value to sampleInitial argument', () => {
+			const sampleInitial = 'dog';
+			const result = makeString(sampleInitial, MOCK_FALLBACK_DEFAULT);
+			expect(result()).toBe(sampleInitial);
+		});
+
+		it('should return fallback default when value is null', () => {
+			const sampleFallback = 'cat';
+			const result = makeString(null, sampleFallback);
+			expect(result()).toBe(sampleFallback);
+		});
+
+		it('should set value when called with non-null value', () => {
```

Comment:
The method makeString should only set the value if it is called with a string. Null and undefined should not set the value, but numbers and booleans should also fail to set the value.